src/SDKs/Batch/Support/FileConventions/Src/AzureBatchFileConventions/AzureBatchFileConventions.csproj

```diff
@@ -1,126 +1,44 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
+﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
-    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
-    <ProjectGuid>{A7F7B253-B796-4DB1-8C51-022B3F31F900}</ProjectGuid>
-    <OutputType>Library</OutputType>
-    <AppDesignerFolder>Properties</AppDesignerFolder>
-    <RootNamespace>Microsoft.Azure.Batch.Conventions.Files</RootNamespace>
+    <PackageId>Microsoft.Azure.Batch.Conventions.Files</PackageId>
+    <Description>
+This library encapsulates a set of storage conventions for Azure Batch task
+and job outputs, making it convenient for clients to browse and retrieve outputs
+by task and job id and the type of output.
+    </Description>
+    <AssemblyTitle>Microsoft Azure Batch File Conventions</AssemblyTitle>
     <AssemblyName>Microsoft.Azure.Batch.Conventions.Files</AssemblyName>
-    <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
-    <FileAlignment>512</FileAlignment>
-    <TargetFrameworkProfile />
-    <AutoRestProjects>true</AutoRestProjects>
-  </PropertyGroup>
-  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
-    <DebugSymbols>true</DebugSymbols>
-    <DebugType>full</DebugType>
-    <Optimize>false</Optimize>
-    <OutputPath>bin\Debug\</OutputPath>
-    <DefineConstants>DEBUG;TRACE</DefineConstants>
-    <ErrorReport>prompt</ErrorReport>
-    <WarningLevel>4</WarningLevel>
+    <VersionPrefix>3.0.0</VersionPrefix>
+    <PackageTags>Microsoft, Azure, Batch, windowsazureofficial</PackageTags>
+    <PackageReleaseNotes>
+Updated dependent libraries.
+    </PackageReleaseNotes>
+    <GeneratePackageOnBuild>True</GeneratePackageOnBuild>
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
   </PropertyGroup>
-  <Import Project="..\..\..\..\..\tools\Library.Settings.targets" />
-  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|AnyCPU' ">
-    <DebugType>pdbonly</DebugType>
-    <Optimize>true</Optimize>
-    <OutputPath>bin\Release\</OutputPath>
-    <DefineConstants>TRACE</DefineConstants>
-    <ErrorReport>prompt</ErrorReport>
-    <WarningLevel>4</WarningLevel>
-    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
-    <DocumentationFile>bin\Release\Microsoft.Azure.Batch.Conventions.Files.XML</DocumentationFile>
+  <PropertyGroup>
+    <SignAssembly>false</SignAssembly>
```

Comment:
@matthchr ideally we highly discourage not being part of the common build properties.
But as this project is a special case (from what I got to know from our offline conversation)
By default the two properties (SignAssembly, DelaySign) are false, so if your intent is to document explicitly the values, then leave it, else you don't need these properties set for 'False'